ably/README.md

```diff
@@ -0,0 +1,65 @@
+## Overview
+The [Ably][1] platform is used to power realtime use cases such as multiplayer, chat, data synchronisation, data broadcast and notifications, for high-scale web and mobile applications around the world.
+
+Using our APIs, engineers are free to focus on building core functionality, rather than having to provision and maintain servers and cloud infrastructure.
+
+Using Ably’s Datadog Integration, you can:
+- Monitor realtime application metrics across your Ably applications
+- Track messages, channels and connection usage
+- Identify unexpected activity and troubleshoot potential issues
+
+
+## Setup
+
+- **In Datadog**: Go to **Integrations**, select the Ably tile and click **Install Integration**.
+
+- **In Ably**: Go to https://ably.com, login and navigate to **Your Apps**.<br/>
+
+![Ably Screenshot][2]
+
+- Select the **Ably App** you would like to setup the **Datadog Integration** for and click **Integrations**.<br />
+
+![Ably Screenshot][3]
+
+- Click the **Connect to Datadog** button. <br />
+
+![Ably Screenshot][4]
+
+Your Ably App statistics data will now appear in Datadog.
+
+## Uninstallation
+
+- **In Ably**: Go to https://ably.com, login and navigate to **Your Apps**.<br/>
+
+- Select the Ably App you would like to uninstall the **Datadog Integration** for.<br />
+
+- Click the **Remove** button in the vDatadog Integration** section.<br />
```

Comment:
```suggestion
- **In Ably**: Go to https://ably.com, login and navigate to **Your Apps**.

- Select the Ably App you would like to uninstall the **Datadog Integration** for.

- Click the **Remove** button in the vDatadog Integration** section.
```

Reply:
Updated in e20e3b965b44b64683573fa2cb3ca92045d2e600